legacy/core/src/main/resources/META-INF/registryFile.jai

```diff
@@ -87,6 +87,9 @@ descriptor	org.eclipse.imagen.operator.SubtractConstDescriptor
 descriptor	org.eclipse.imagen.operator.ClampDescriptor
 descriptor	org.eclipse.imagen.operator.ColorConvertDescriptor
 descriptor	org.eclipse.imagen.operator.ConvolveDescriptor
+descriptor	org.eclipse.imagen.operator.CropDescriptor
```

Comment:
I still see references in org.eclipse.imagen.media.opimage.properties
that have been commented out for other operators (e.g., Affine).